modules/lexiconIdSystem.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * This module adds Lexicon to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/lexiconIdSystem
+ * @requires module:modules/userId
+ */
+
+import { logError } from '../src/utils.js';
+import * as ajax from '../src/ajax.js';
```

Comment:
I think the Prebid mantainers that modules import only the functions they use instead of doing `import *`.